### PR TITLE
Feanil/run role vagrant file

### PR DESCRIPTION
**Pull request by feanil**

### CODE REVIEW

**Comment by benpatterson** [comment]
@feanil I was holding off on adding any motd.d changes. I figured I could put that in a subsequent PR (in part b/c of some discussion of implementation (i.e. genericize it to 'all vagrant' or evaluate situations individually). 

That said, I'm also happy to add those changes to this branch if that's preferable. 

CC @hkim823

**Comment by feanil** [comment]
@benpatterson I've been trying to get someone to review this but that hasn't happened yet.  If you can review it and it looks good to you, I'd be happy to merge this.

File: vagrant/base/test_role/Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    # Allow DNS to work for Ubuntu 12.10 host
+    # http://askubuntu.com/questions/238040/how-do-i-fix-name-service-for-vagrant-client
+    vb.customize ["modifyvm", :id, "--natdnshostresolver1", "on"]
+  end
+
+  config.vm.provision :ansible do |ansible|
+    # point Vagrant at the location of your playbook you want to run
+    ansible.playbook = "../../../playbooks/run_role.yml"
+    ansible.verbose = "extra"
+    ansible.extra_vars = { role: ENV['VAGRANT_ANSIBLE_ROLE'] }
```
Comment (benpatterson) [inline vagrant/base/test_role/Vagrantfile:26]: Should there be a bit of logic to fail early when this var is not set?

**Comment by benpatterson** [comment]
I had a question but :thumbsup: either way.

File: vagrant/base/test_role/Vagrantfile (hunk 1 above)
Comment (feanil) [inline vagrant/base/test_role/Vagrantfile:26]: That's not a bad idea but I'm not gonna worry about that for now.

**Pull request merged**